docs/plugins/ingest.asciidoc

```diff
@@ -25,6 +25,13 @@ under the CCA-ShareAlike 3.0 license. For more details see, http://dev.maxmind.c
 
 A processor that extracts details from the User-Agent header value.
 
+[float]
+=== Community contributed repository plugins
```

Comment:
I think this should be "ingest plugins" in this case

Reply:
Ah, good catch...